tests/app/test_retriever.py

```diff
@@ -143,7 +145,7 @@ def create_threads(self, no_of_threads=1, add_internal_draft=False, add_responde
             msg_id = str(uuid.uuid4())
             thread_id = msg_id
             threads.append(thread_id)
-            self.add_secure_message(msg_id=msg_id, thread_id=thread_id, survey=constants.BRES_USER, from_internal=False)
+            self.add_secure_message(msg_id=msg_id, thread_id=thread_id, survey=test_utilities.BRES_SURVEY, from_internal=False)
```

Comment:
Unsure what's happening here: 
`survey=constants.BRES_USER` = "BRES"
`survey=test_utilities.BRES_SURVEY` = "33333333-22222-3333-4444-88dc018a1a4c"

Unless I'm missing something?

Reply:
Yea... this seems pretty weird to me too

Reply:
This was changed because before it was using the name "BRES" and changing this to the non specific user wouldn't make sense as it means "GROUP" so we thought having an actual survey there would be better